frontend/src/app/blog/[slug]/page.tsx

```
@@ -0,0 +1,344 @@
+                  </p>
+                )}
+              </div>
+            </div>
+          </div>
+        )}
+
+        {/* Content */}
+        <div className="container mx-auto px-4 py-8">
+          <div className="max-w-4xl mx-auto">
+            <div className="prose prose-lg max-w-none">
+              <div dangerouslySetInnerHTML={{ __html: post.contentHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)